src/joint.dia.graph.js

```diff
@@ -75,6 +63,17 @@ joint.dia.Graph = Backbone.Model.extend({
         // to the outside world.
         cells.on('all', this.trigger, this);
 
+        this.on('batch:start', this._batchStarted, this);
+        this.on('batch:stop', this._batchStopped, this);
+        // Backbone automatically doesn't trigger re-sort if models attributes are changed later when
+        // they're already in the collection. Therefore, we're triggering sort manually here.
+        this.on('change:z', function(e) {
+            return !this.hasActiveBatch('to-front') && !this.hasActiveBatch('to-back') && cells.sort();
```

Comment:
Handlers does not have to return a value + I would rather not to use constructs like `a && b()` but `if (a) { b() }` instead
```javascript
if (!this.hasActiveBatch('to-front') && !this.hasActiveBatch('to-back')) {
  cells.sort();
}
```